admin/datasource.ts

```diff
@@ -55,7 +53,9 @@ export const ADMINJS_ENTITIES = [
   LongTermProjectOperating,
   MRV,
   SequestrationRate,
+  EcosystemExtent,
   Project,
+  BaseDataView,
```

Comment:
I don't think we need `BaseDataView` injected into AdminJS's TypeORM adapter, as in, for sure we won't need it as a Resource.

But maybe it has to be here for the whole thing to work?